SURVEY.md

```diff
@@ -4,121 +4,53 @@
+So lets participate in a survey which will help understand us better. The main aim of the survey is to get a better understanding of ourselves like who we are, where are we from, what  we are doing , do we need a scholarship  ,if yes, why etc
 
 
-Please add the questions that you feel can be included in the survey. 
+Please add the questions that you feel can be included in the survey.
 (feel free to edit the content given below :)  )
 
 Add all the questions that pops in your mind we will rank them later on.**
 
-### 1.What do you use Pandas for?(multiple choice /single choice)
-- Data analysis
-- Machine learning
-- Programming of web parsers / scrapers / crawlers
-- Software testing / Writing automated tests
-- Network programming
-- Software prototyping
-- DevOps / System administration / Writing automation scripts
+## 1. Demographics
 
+### 1.1. What is your age range?
+- 17 or younger
+- 18–20
+- 21–29
+- 30–39
+- 40–49
+- 50–59
+- 60 or older
 
-### Which of pandas capabilities are most important to you?
+### 1.2. What is your country of origin?
+- (open question)
 
-- Loading Data
-- Rename Function
-- Map
-- Apply and Apply Map
-- Shape and Columns
-- Unique function
-- Cross Tab
-- Accessing Sub data frames
-- Plotting
-- Groupby and Statistics
-- Aggregate function
-- Join
-- Masking
-- Handling Missing Values
+### 1.3. What is your country of residence?
+- (open question)
 
-### Do you use pandas at work?
-### How would you rate pandas? (Very Satisfied = 5 Satisfied = 4 Neutral = 3 Dissatisfied = 2 Very Dissatisfied = 1)
+## 2. Employment information
 
-- Accuracy
-- Documentation
-- Reliability and stability
-- Usability 
-
-### Employment status
-
-- Fully employed by a company / organization
-- Student
-- Freelancer
+### 2.1. What is your current employment status?
+- Fully employed by a company/organization
+- Partially employed by a company/organization
 - Self-employed
-- Partially employed by a company / organization
+- Freelancer
+- Student
 - Retired
 - Other
 
-### Job Roles
-
-- Developer / Programmer Architect
-- Team Lead
-- Data Analyst
-- Data Scientist
-- Technical Support
-- Systems Analyst
-- CIO / CEO / CTO
-- QA Engineer
-- Product Manager
-- Business Analyst
-- DBA
-- Other
-
-### Working in a Team vs Working Independently
-
-- Work in a team
-- Work on your own project(s)independently
-- Work as an external consultant or trainer
-
-### Team Size
-
-- 2–7 people
-- 8–12 people
-- 13–20 people
-- 21–40 people
-- More than 40 people
-
-### Company Size
+### 2.2. What is your job role?
+- (open question)
 
-- Just me
-- 2–10
-- 11–50
-- 51–500
-- 501–1,000
-- 1,001–5,000
-- More than 5,000
-- Not sure
-
-### Experience in IT
+### 2.3. For how long have you kept this job role?
```

Comment:
For my interests, I find more relevant the previous question or total years of experience (with pandas, python or in IT in general). Happy to also have this, but I'd like to have the others.

And I'm not sure if it's useful to have bins, all the questions with numerical values (age, years of experience...) can probably ask for the exact value.

Reply:
Ok, makes sense! I will re-add the deleted questions. I also agree with the age values, I can change the output of these questions as well.